website/content/en/docs/provisioner-crd.md

```diff
@@ -141,7 +141,7 @@ Karpenter supports specifying capacity type, which is analogous to [EC2 purchase
 
 This section is cloud provider specific. Reference the appropriate documentation:
 
-- [AWS](../aws/constraints/)
```

Comment:
For parity, we might want to name this file provisioner.md.

Reply:
Good idea. Done. https://github.com/aws/karpenter/pull/950